packages/starlight/utils/git.ts

```diff
@@ -105,7 +105,9 @@ export function getAllNewestCommitDate(directory: string): [string, number][] {
 
 	return Array.from(latestDates.entries()).map(([file, date]) => {
 		const fileFullPath = resolve(repoRoot, file);
-		const fileInDirectory = relative(directory, fileFullPath);
+		let fileInDirectory = relative(rootPath, fileFullPath);
+		// Format path to unix style path.
+		fileInDirectory = fileInDirectory?.replace(/\\/g, '/');
```

Comment:
Is this an Astro v5-related change? Or a bug fix for an existing issue?

Reply:
This is an Astro v5-related change related to the fact that we now resolve from `rootPath` and without that change our tests started failing. If you want a detailed breakdown on exactly why, it'll have to wait when I get access to a Windows machine and examine the differences.